### PR TITLE
Fixed update_server_build to allow backups

### PR DESCRIPTION
**Pull request by jozefbonnar**
This now allows you to update_server_build without throwing the error or not having the backups field present

### CODE REVIEW

**Comment by jozefbonnar** [comment]
@iamkubi

File: pydactyl/api/servers.py [hunk 1]
```diff
@@ -300,10 +300,12 @@ def update_server_build(self, server_id, allocation_id, memory_limit=None,
...
 
         Example of a working set of parameters:
-            update_server_build(server_id=12, allocation_id=1964,
-                    memory_limit=8001, swap_limit=0, disk_limit=30000,
-                    cpu_limit=200, io_limit=500, database_limit=0,
-                    allocation_limit=0, oom_disabled=True)
+            update_server_build(server_id=12, allocation_id=81, 
+                    memory_limit=2048, swap_limit=2048, disk_limit=5120, 
+                    cpu_limit=100, io_limit=500, database_limit=1, 
+                    allocation_limit=2, backup_limt=4,
```
Comment (iamkubi) [inline pydactyl/api/servers.py:306]: limit is misspelled here

**Comment by iamkubi** [comment]
Please fix the typo, then LGTM

**Comment by jozefbonnar** [comment]
@iamkubi fixed

**Pull request merged**